pennylane/devices/qubit/simulate.py

```diff
@@ -305,6 +304,8 @@ def simulate_native_mcm(
         one_shot_meas, mcm_values_dict = simulate_one_shot_native_mcm(
             aux_circuit, rng, prng_key, debugger, interface
         )
+        if one_shot_meas is None:
```

Comment:
What would give rise to `one_shot_meas` being `None`?

Reply:
This comes out `None` when we post-select a zero-prob branch of the statevector. It is just a convention allowing to know when to discard the shot.